src/ca/team2706/vision/trackerboxreloaded/Main.java

```diff
@@ -41,13 +46,52 @@
 
 	/**
 	 * A class to hold any vision data returned by process()
+	 * :)
+	 * :)
+	 * :}
+	 * :]
+	 * :]
 	 */
 	public static class VisionData {
 		public Mat outputImg = new Mat();
-		double fps;
+		public double fps;
+		public HashMap<String,String> data = new HashMap<String,String>();
+		/**
+		 * This method converts the vision data into a nice and tidy string
+		 * :]
+		 * @return the data
+		 */
+		public String encode(NetworkTable table){
+			String s = DataUtils.encodeData(data);
+			return s;
+		}
+		
+		/**
+		 * Decodes the NetworkTable stuff into a VisionData
+		 * @param the data table
+		 * @return the data
+		 */
+		public static VisionData decode(NetworkTable table){
+			
+			return null;
+		}
 	}
-
-	/*** Helper Functions ***/
+	/**
+	 * Initilizes the Network Tables
+	 */
+	private static void initNetworkTables(){
+		NetworkTableInstance instance = NetworkTableInstance.getDefault();
```

Comment:
This seems like you might want to put a warning to change these values when we go to use it